### DISCUSSION TITLE
No option to download beta3?

### DISCUSSION THREAD

**Discussion by davidenco** [snapappointments/bootstrap-select#2725] [Q&A]
I noticed you bumped to beta3 about 5 months ago but the latest release is still showing as beta2. Can you provide a download link for beta3 please, or alternatively provide instructions on how to manually compile beta3 myself? Thanks.

**Comment by NicolasCARPi** [comment]
It's only a draft release, that's why. @caseyjhol why are you still holding on this release? From my point of view beta versions can be released even if they are missing some bits, after all, they are beta!
